src/main/java/com/thebois/Pawntastic.java

```diff
@@ -147,7 +147,7 @@ private void createModels() throws IOException, ClassNotFoundException {
             world = new World(WORLD_SIZE, 0);
             colony = new Colony(world.findEmptyPositions(PAWN_POSITIONS),
```

Comment:
Thoughts on adding a constructor to Colony that doesn't take in a list of positions. Meaning it instantiates a colony with no pawns.

Reply:
Im for it, but I remember we talked about this earlier and for some reason we decried against it. Think the counter argument was that pawns belong in the colony.

Reply:
The constructor now takes an `IPositionFinder` and an `EventBus`.